frontend/.eslintrc.cjs

```diff
@@ -15,7 +15,7 @@ module.exports = {
   rules: {
     "no-unused-vars": "off", // or "@typescript-eslint/no-unused-vars": "off",
     "unused-imports/no-unused-imports": "error",
-    "unused-imports/no-unused-vars": [
+    "@typescript-eslint/no-unused-vars": [
```

Comment:
Not sure what this rule change does, could you elaborate?

Reply:
It was for the type too, I've changed it back!